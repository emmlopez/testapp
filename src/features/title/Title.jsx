import React from "react";
import PropTypes from "prop-types";
import "./TitleH3.css";
import "./TitleH4.scss";

const propTypes = {
  subtitle: PropTypes.string,
  title: PropTypes.string,
};
/**
 * The title component
 * @param {string}  title       The main title
 * @param {string}  subtitle    The subtitle
 * @constructor
 */
const Title = ({ title, subtitle }) => {
  return (
    <>
      <h3 className={"greenH3"}>{title}</h3>
      <h4 className={"redH4"}>{subtitle}</h4>
    </>
  );
};
Title.propTypes = propTypes;
export default Title;
