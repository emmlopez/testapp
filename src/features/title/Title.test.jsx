import React from "react";
import { shallow } from "enzyme";
import Title from "./Title";

describe("Tests for Title component", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(
      <Title title={"The Title"} subtitle={"The SubTitle"} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
