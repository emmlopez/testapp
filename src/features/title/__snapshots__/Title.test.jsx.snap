// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Title component should match snapshot 1`] = `
<Fragment>
  <h3
    className="greenH3"
  >
    The Title
  </h3>
  <h4
    className="redH4"
  >
    The SubTitle
  </h4>
</Fragment>
`;
